package.json

```diff
@@ -87,7 +87,7 @@
     "livekit-client": "^2.5.7",
     "lodash": "^4.17.21",
     "loglevel": "^1.9.1",
-    "matrix-js-sdk": "matrix-org/matrix-js-sdk#8e9a04cdec0f88fc876bbbf406db55b0677f005d",
+    "matrix-js-sdk": "matrix-org/matrix-js-sdk#0098c1a37d33123db5e5c54c6292a9cca7a87ca5",
```

Comment:
You can drop my change that just landed too :) #2764

Reply:
sure. otherwise its not really testable right?